styles/spotify-web/catppuccin.user.css

```diff
@@ -295,6 +298,82 @@
         background-color: @crust;
       }
     }
+    // Cookie Banner
+    #onetrust-banner-sdk {
+        background-color: @base !important;
+        color: @text !important;
+    }
+    #onetrust-policy-text, .ot-dpd-title, .onetrust-policy-title, .ot-text-bold {
+        color: @text !important;
+    }
+    .ot-dpd-desc, .ot-link-btn {
+        color: @text !important;
+    }
+    #onetrust-consent-sdk #onetrust-policy-title {
+        color: @text !important;
+    }
+    #onetrust-banner-sdk button {
+        color: @text !important;
+    }
+    #onetrust-pc-btn-handler {
+          background-color: @base !important;
+    }
+    // Cookie Settings
+      #onetrust-consent-sdk #onetrust-pc-sdk,
+      .ot-acc-txt, .ot-acc-grpdesc{
+          background-color: @base !important;
+      }
+      #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-title, 
+      #onetrust-consent-sdk #onetrust-pc-sdk #ot-pc-desc,
+      #onetrust-consent-sdk #onetrust-pc-sdk h3,
+      #onetrust-consent-sdk #onetrust-pc-sdk h5,
+      #onetrust-consent-sdk #onetrust-pc-sdk h4,
+      #onetrust-consent-sdk #onetrust-pc-sdk h6,
+      #onetrust-consent-sdk #onetrust-pc-sdk h2,
+      #onetrust-pc-sdk .ot-always-active,
+      .ot-acc-txt, .ot-acc-grpdesc,
+      #onetrust-consent-sdk #onetrust-pc-sdk p{
+          color: @text !important;
+      }
+      #onetrust-pc-sdk .ot-accordion-layout.ot-cat-item {
+          border-color: @accent-color !important;
+      }
+      #onetrust-consent-sdk #onetrust-pc-sdk button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler):not([aria-expanded]):not(.ot-link-btn) {
+          background-color: @accent-color !important;
+          border-color: @accent-color !important;
+      }
+      #onetrust-pc-sdk .ot-accordion-layout.ot-cat-item {
+          border-radius: 8px;
+      }
+    // Vendors List
+      #onetrust-consent-sdk #onetrust-pc-sdk .ot-sel-all-hdr span {
+          color: @text;
+      }
+      #onetrust-pc-sdk input[type=text] {
+          background-color: @mantle !important;
+          border-color: @accent-color !important;
+      }
+      #onetrust-pc-sdk .ot-pc-header,
+      #onetrust-pc-sdk ul li{
+          border-bottom-color: @accent-color !important;
+          border-top-color: @accent-color !important;
+      }
+      #ot-ven-lst {
+          border-top-color: @accent-color !important;
+      }
+      .ot-pc-footer {
+          border-top-color: @accent-color !important;
+      }
+      #onetrust-pc-sdk li>button {
+          border-top-color: @accent-color !important;
+      }
+      #onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk {
+          background-color: @base !important;
+      }
```

Comment:
cc @isabelroses I actually themed this for Stack Overflow (https://github.com/catppuccin/userstyles/blob/c977d9e354082ca79e78bc7bec04da0ac2611e41/styles/stack-overflow/catppuccin.user.css#L313-L406), seeing as it looks like a reusable component from across websites maybe we add a global userstyle that includes tweaks like this? Or possibly generate it statically and add `@import` lines for userstyles that involve websites with the OneTrust cookie consent popup?

Reply:
Ohhh this is interesting. Heres my idea. We create a "common" dir and place commonalities between themes and publish them there. But I'll admit this maybe a bit overkill and just repeating the same style here might be easier.